manifest.yml

```diff
@@ -13,6 +13,7 @@ applications:
       OPTIMIZE_MEMORY: true
       EASEY_DB_SERVICE: ((dbSvc))
       EASEY_MONITOR_PLAN_API_HOST: ((host))
+      EASEY_MONITOR_PLAN_API_REPLICA_HOST: ((replicaHost))
```

Comment:
This environment variable is unnecessary, and the `replicaHost` is not set in the `-vars` files.
```suggestion
```